### PR TITLE
Fix GitLogParser dialogues and docs

### PR DESCRIPTION
**Pull request by MW-Friedrich**
# Fix GitLogParser dialogues and docs

Closes: #3078 

## Description

Some uncaught regressions were introduced in the new subcommands of the GitLogParser and #3041 , which are now fixed and covered by tests. There are also updates to the documentation to make clearer what the subcommands do internally. Formatting is also enhanced.

### CODE REVIEW

**Comment by ce-bo** [comment]
In such cases, I would not reopen the issue. I would rather open a new bug issue.

**Comment by ce-bo** [comment]
Is it still a draft PR?

File: analysis/import/GitLogParser/README.md [hunk 1]
```diff
@@ -81,17 +68,25 @@ The result is written as JSON to standard out or into an output file (if specifi
...
 The resulting project has the project name specified for the GitLogParser.
 
-## Examples
+#### Creating the repository log for metric generation
+
+| SCM | Log format                   | Command for log creation                            | tracks renames | ignores deleted files | supports code churn |
+| --- | ---------------------------- | --------------------------------------------------- | -------------- | --------------------- | ------------------- |
+| git | GIT_LOG_NUMSTAT_RAW_REVERSED | `git log --numstat --raw --topo-order --reverse -m` | yes            | yes                   | yes                 |
+
+You can also use the bash script anongit which generates an anonymous git log with log format GIT_LOG for usage with
```
Comment (ce-bo) [inline analysis/import/GitLogParser/README.md:77]: Let's add a link to the anongit file.

**Review by ce-bo — APPROVED** [review]

**Comment by ce-bo** [comment]
LGTM. Thanks for the improvements!

**Comment by MW-Friedrich** [comment]
> Is it still a draft PR?

It was, since I was waiting for your feedback, but now I'm done. Thanks for the review!

**Pull request merged**